addons/website_hr_recruitment/controllers/main.py

```diff
@@ -5,6 +5,7 @@
 from odoo.addons.http_routing.models.ir_http import slug
 from odoo.osv.expression import AND
 from odoo.http import request
+from odoo.tools.misc import groupby
```

Comment:
Commit title: `[IMP] website_hr_recruitment: add filters` ?
Also, it would be nicer if the template updates related to the filters were introduced inside this commit.

Reply:
Indeed, done 👍